tools/msvs/msi/product.wxs

```diff
@@ -110,24 +103,18 @@
 
     <Directory Id="TARGETDIR" Name="SourceDir">
       <Directory Id="ProgramMenuFolder">
-        <Directory Id="ApplicationProgramsFolder" Name="io.js"/>
+        <Directory Id="ApplicationProgramsFolder" Name="Node.js"/>
       </Directory>
 
       <Directory Id="$(var.ProgramFilesFolderId)">
-        <Directory Id="INSTALLDIR" Name="iojs">
+        <Directory Id="INSTALLDIR" Name="nodejs">
```

Comment:
I suggested `nodejs` in an outdated diff somewhere up in this thread, just because that's what's in v0.x. But no strong feelings about it.